apps/dashboard/app/models/project.rb

```diff
@@ -27,33 +27,69 @@ def find(project_path)
     def dataroot
       Rails.logger.debug("project path is: #{OodAppkit.dataroot.join('projects')}")
 
-      OodAppkit.dataroot.join('projects').tap do |p|
+      OodAppkit.dataroot.join('projects').tap do |path|
         p.mkpath unless p.exist?
```

Comment:
You've changed this variable to `path` but then didn't change what's used in the block.

Reply:
Facepalm. Thanks!